tests/python_tests/test_continuous_batching.py

```diff
@@ -74,11 +74,11 @@ def test_e2e_real_models(tmp_path, model_id):
 @pytest.mark.parametrize("generation_config", test_configs)
 @pytest.mark.parametrize("prompt", batched_prompts[1:])  # num_beams=15 diverges on the first prompt.
 @pytest.mark.precommit
-def test_continuous_batching_vs_stateful(prompt, generation_config):
+def test_continuous_batching_vs_stateful(prompt, generation_config):  # TODO check this test
```

Comment:
Describe TODO more clearly or apply

Reply:
I already checked that test, but forget to delete comment. Will remove it